rosco-web/config/rosco.yml

```diff
@@ -105,47 +105,47 @@ aws:
       - region: us-east-1
         virtualizationType: hvm
         instanceType: t2.micro
-        sourceAmi: ami-9eaa1cf6
+        sourceAmi: ami-9d751ee7
```

Comment:
Could you link to where you got these as comments for the various ami IDs? This will affect all rosco users who are depending on the default configuration & it would be nice for them to easily verify (in the config) what they're using as a base image.

Reply:
Sure thing. Update the diff with the link.